docs/source/virtual_rainforest/soil/soil_details.md

```diff
@@ -50,7 +50,7 @@ Another mechanism by which carbon is protected from microbial activity is minera
 association, whereby mineral surfaces take up organic matter by adsorption, conferring
 chemical protection. It is generally microbial biomass and LMWC that gets absorbed in
 this way. Aggregates can form from MAOM, which they then release upon breakdown. When
-desorption occurs carbon is released from mineral associated in the form of LMWC.
+desorption occurs carbon is released from mineral association in the form of LMWC.
```

Comment:
Is "mineral assocation" a domain-specific term?

Reply:
Yeah, I think it is. In soil science it refers to carbon bound to soil minerals which is therefore not accessible for use by soil microbes.

Just for clarity I added a bunch of small doc fixes while I was working on the command line docs (not sure if that is best practice?), so this bit forms part of the soil module specific docs, rather than the command line docs.

Reply:
Ok cool. That's what I thought, but I just wanted to check that it made sense to someone!

I think it's fine to mix in various fixes on the same PR, although generally you want a theme -- and "misc doc fixes" is fine as a theme. I would usually do them as separate commits though for revertability.